assets/src/components/template-inserter/block-preview.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { createBlock } from '@wordpress/blocks';
+import { Disabled } from '@wordpress/components';
+// import { BlockEdit } from '@wordpress/block-editor';
+
+/**
+ * Block Preview Component: It renders a preview given a block name and attributes.
+ *
+ * @param {Object} props Component props.
+ *
+ * @return {WPElement} Rendered element.
+ */
+function BlockPreview( props ) {
+	return (
+		<button onClick={ props.onClick } className="components-button editor-block-preview block-editor-block-preview">
+			<BlockPreviewContent { ...props } />
+		</button>
+	);
+}
+
+export function BlockPreviewContent( { name, attributes } ) {
+	// @todo Importing this outside of the function causes error for some reason.
```

Comment:
This is because the new block-editor package is not yet listed in `wpDependencies` in `webpack.config.js` nor in `package.json`.

You can still use `@wordpress/editor` for now. It just proxies to the new package behind the scenes for BC.

Alternatively, since I am probably going to merge #2000 today and doing some rewriting there from `@wordpress/editor` to `@wordpress/block-editor`, you could then just merge in the latest changes from the amp-stories-redux branch.